scripts/query-endpoint-resources.sh

```diff
@@ -20,6 +20,13 @@ jq -c '.[]' EndpointResourcesList.json | while read endpoint; do
          cd ../../../resources/prod_resources
       else
          curl -s -o $FILENAME $URL
+         
+         if [ "$NAME" = "Cerner" ]
```

Comment:
What does this do?

Reply:
Cerner is the one list that has the endpoints list in a different format, it is a lower case e in endpoints rather than upper case e in Endpoints, thus the delete function below to remove all but 10 lines is different depending on if the file is Cerner or not

Reply:
ah okay got it, thanks!